src/osd/PGLog.h

```diff
@@ -899,7 +899,6 @@ struct PGLog : DoutPrefixProvider {
     const mempool::osd_pglog::list<pg_log_entry_t> &orig_entries, ///< [in] entries for hoid to merge
     const pg_info_t &info,              ///< [in] info for merging entries
     eversion_t olog_can_rollback_to,     ///< [in] rollback boundary
```

Comment:
rollback boundary of what? let's improve this comment since this variable now represents different things